src/screens/CreatePost/index.js

```diff
@@ -448,7 +446,7 @@ const CreatePost = () => {
   const checkTaggingUser = () => {
     const mapTagUser = taggingUsers.map((data) => {
       const findData = allTaggingUser.find((dataUser) => dataUser.username === data);
-      return findData.user_id;
+      return findData?.user_id;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is good for preventing potential errors when `findData` is undefined. It ensures that the function doesn't try to access the `user_id` property of `undefined`, which would throw an error.



<!-- This is an auto-generated comment by OSS CodeRabbit -->